.github/workflows/validation.yml

```diff
@@ -50,20 +56,33 @@ jobs:
           make validate-os-android assert-clean
       - run: make clean
 
-  validate:
+  validate-os:
```

Comment:
at this juncture we might was well rename it to `validate-core`, WDYT?

Reply:
Sure, I just kept os because it's the command in the Makefile — `make validate-os`